ca_description/urdf/create_base.xacro

```diff
@@ -51,7 +51,12 @@
       <origin xyz="0 -0.04 0.13" rpy="0 0 0"/>
     </xacro:imu_sensor>
 
-    <xacro:create_apriltag/>
+    <!--Setting Apriltag Variables-->
+    <xacro:property name="apriltag_origin" >
```

Comment:
This is bad. You have to apply exactly the same as you did for the color sensors.

Reply:
Also, this variable is being used in another file. It's very hard to track errors in this way.